integration-cli/docker_utils.go

```diff
@@ -502,10 +502,9 @@ func init() {
 	// Obtain the daemon platform so that it can be used by tests to make
 	// intelligent decisions about how to configure themselves, and validate
 	// that the target platform is valid.
-	res, b, err := sockRequestRaw("GET", "/version", nil, "application/json")
-	defer b.Close()
```

Comment:
Don't think we need it at all - it will automatically close when goes out of scope.